python/assistant-stream/src/assistant_stream/modules/tool_call.py

```
@@ -49,6 +49,10 @@ def set_result(self, result: Any):
             result=result,
         )
         self.loop.call_soon_threadsafe(self.queue.put_nowait, chunk)
+        self.close()
+
+    def close(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `close`

Add return type annotation: `None`

(ANN201)